mlflow/R/mlflow/R/model-mleap.R

```diff
@@ -0,0 +1,53 @@
+#' @include model-utils.R
+NULL
+
+#' @rdname mlflow_save_model
+#' @param sample_input Sample Spark DataFrame input that the model can evaluate. This is required by MLeap for data schema inference.
+#'
+#' @export
+mlflow_save_model.ml_pipeline_model <- function(model,
+                                                path,
+                                                model_spec = list(),
+                                                conda_env = NULL,
+                                                sample_input = NULL,
+                                                ...) {
+  if (is.null(sample_input)) {
+    stop("`sample_input` is required by MLeap for data schema inference.")
+  }
+
+  assert_pkg_installed("mleap")
+  if (dir.exists(path)) unlink(path, recursive = TRUE)
+  dir.create(path)
+
+  model_filename <- "model.zip"
```

Comment:
Quick check did not give me an answer, so: Is model.zip the file name currently used with MLeap models in mlflow? Also, at some point, it might be good to keep model names for all flavors in a variable in the R mlflow namespace instead of defining them ad-hoc once on saving and once on loading.

Reply:
For MLeap currently it just needs to be any file name ending with `.zip`